pom.xml

```diff
@@ -140,6 +140,14 @@
                 <configuration>
                     <configLocation>/config/checkstyle.xml</configLocation>
                 </configuration>
+                <executions>
+                    <execution>
+                        <id>checkstyle</id>
+                        <goals>
+                            <goal>check</goal>
```

Comment:
Default phase is "verify", and it's already too late. Checkstyle should be run with compilation

Reply:
Verification added